ZCRMiOS/Classes/MetaDataAPIHandler.swift

```diff
@@ -118,15 +118,7 @@ internal class MetaDataAPIHandler : CommonAPIHandler
         module.setArguments(arguments: moduleDetails.optArrayOfDictionaries(key: "arguments"))
         if(moduleDetails.hasValue(forKey: "$properties"))
         {
-            let dollarProperties = moduleDetails.optArray(key: "$properties") as! [String]
-            var properties : [String] = [String]()
-            for dollarProperty in dollarProperties
-            {
-                var property = dollarProperty
-                property.removeFirst()
-                properties.append(property)
-            }
-            module.setProperties(properties: properties)
+            module.setProperties(properties: moduleDetails.optArray(key: "$properties") as! [String])
```

Comment:
Is this change new?

Reply:
As you said, replacing $ in $properties is not needed, code for replacing is removed